admin/sales_analytics/metrics.py

```diff
@@ -0,0 +1,113 @@
+from admin.base.settings import ENTRY_POINTS
+from admin.sales_analytics.utils import get_entry_point, get_sorted_index
+from datetime import datetime, timedelta
+from framework.mongo import database as db
+
+
+def get_user_count(db=db, entry_points=ENTRY_POINTS):
+    """
+    Get the number of users created from each entry point: osf, osf4m, prereg, and institution.
+    """
+    count_list = []
+    percent_list = []
+    tags = ['osf4m', 'prereg', 'institution', 'osf']
```

Comment:
Good points about updating the list. I didn't have tags in the beginning, but names in the ENTRY_POINTS are not good for labelling the data. so maybe changing ENTRY_POINTS to a list of tuples that contains the original system tags and their display names?